easybuild/easyconfigs/a/Automake/Automake-1.15-GCCcore-5.4.0.eb

```diff
@@ -22,11 +10,15 @@ toolchain = {'name': 'GCCcore', 'version': '5.4.0'}
 
 source_urls = [GNU_SOURCE]
 sources = [SOURCELOWER_TAR_GZ]
+checksums = ['7946e945a96e28152ba5a6beb0625ca715c6e32ac55f2e353ef54def0c8ed924']
 
 dependencies = [('Autoconf', '2.69')]
 
 # use same binutils version that was used when building GCCcore toolchain
-builddependencies = [('binutils', '2.26', '', True)]
+builddependencies = [
+    ('binutils', '2.26', '', True),
+    ('Perl', '5.24.0'),
```

Comment:
This is a bit of a PITA, but given the discussion in #1822, I don't see another way out...

Reply:
To me it seems a reasonable fix. Plus it is a build dependency that, if I understand correctly, is used for generating documentation. I do not expect this to cause dependency issues with OS Perl modules.

Reply:
Well, yeah, it's not a huge issue, just a bit annoying that we have to force installing building & `Perl` first before we can reliably install something fairly simple as `Automake`.

Reply:
I agree on that